bom/pom.xml

```diff
@@ -95,6 +95,7 @@ quarkus.smallrye-health.readiness-path=${health.check.readiness-path}
 quarkus.smallrye-health.startup-path=${health.check.startup-path}
 quarkus.vertx.prefer-native-transport=true
 quarkus.vertx.resolver.cache-max-time-to-live=0
+quarkus.log.console.json=false
```

Comment:
can you please move this line up so that it follows the lexical ordering?

Reply:
done :)